src/CallControl.tsx

```diff
@@ -67,23 +70,18 @@ const useStyles = makeStyles((theme) => ({
 const CallControl: React.FC<CallControlProps> = (props) => {
   const classes = useStyles();
 
-  const [micEnabled, setMicEnabled] = useState(true);
```

Comment:
I moved this state to the Conference class to have control over them there, and made the same values props in this class